compiler/src/dotty/tools/dotc/quoted/PickledQuotes.scala

```diff
@@ -91,11 +91,6 @@ object PickledQuotes {
               val quotedType = typeHole(idx, reifiedArgs)
               PickledQuotes.quotedTypeToTree(quotedType)
           }
-        case tree: Select =>
-          // Retain selected members
```

Comment:
Happy to see a bug fixed by deleting code, but I don't really understand what this code was there for in the first place :sweat_smile:. What does retaining means here?

Reply:
I do not remember exactly. There was an issue where the selects lost their symbol and this was a workaround. We must have fixed the actual source of the issue in the meantime.